assets/css/admin.css

```diff
@@ -726,10 +735,10 @@ table.wc_gateways .renewals .tips {
 }
 
 /* Hide irrelevant sections on Edit Subscription screen */
-body.post-type-shop_subscription
-	.order_actions
-	#actions
-	optgroup[label='Resend order emails'],
+body.woocommerce_page_wc-orders--shop_subscription .order_actions #actions optgroup[label='Resend order emails'],
```

Comment:
Although this one-line-per-rule formatting arguably looks neater, I think we should auto-format this file in line with the repo's `.prettierrc` rules.